src/ResourceManager/Resources/Commands.Resources/RoleAssignments/GetAzureRoleAssignmentCommand.cs

```diff
@@ -36,6 +36,7 @@ public class GetAzureRoleAssignmentCommand : ResourcesBaseCmdlet
         [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, ParameterSetName = ParameterSet.ScopeWithObjectId,
             HelpMessage = "The user or group object id.")]
         [ValidateNotNullOrEmpty]
+        [Alias("Id", "PrincipalId")]
```

Comment:
This comes as direct feedback from an exec - we need to make our PowerShell parameters match as much as possible our JSON body.